server/label_linux.go

```diff
@@ -11,12 +12,17 @@ import (
 
 func securityLabel(path, secLabel string, shared, maybeRelabel bool) error {
 	if maybeRelabel {
-		currentLabel, err := label.FileLabel(path)
-		if err == nil && currentLabel == secLabel {
-			logrus.Debugf(
-				"Skipping relabel for %s, as TrySkipVolumeSELinuxLabel is true and the label of the top level of the volume is already correct",
-				path)
-			return nil
+		canonicalSecLabel, err := selinux.CanonicalizeContext(secLabel)
+		if err != nil {
+			logrus.Errorf("Canonicalize label failed %s: %v", secLabel, err)
+		} else {
+			currentLabel, err := label.FileLabel(path)
```

Comment:
should we Canonicalize this one as well, or do we think it'll be correct because we're reading it from disk (after it's been procesed by the kernel)